test/tests/jobs/FinishJobTest.cpp

```diff
@@ -14,12 +15,17 @@ struct FinishJobTest : tpunit::TestFixture {
                               TEST(FinishJobTest::deleteFinishedJobWithNoChildren),
                               TEST(FinishJobTest::hasRepeat),
                               TEST(FinishJobTest::inRunqueuedState),
-                              AFTER(FinishJobTest::tearDown),
+                              TEST(FinishJobTest::hasRepeatWithDelay),
+                              TEST(FinishJobTest::hasDelay),
+                              TEST(FinishJobTest::hasRepeatWithNextRun),
+                              */
+                              TEST(FinishJobTest::hasNextRun),
+                              //AFTER(FinishJobTest::tearDown),
```

Comment:
Why is this all commented out?

Reply:
I did when testing and forgot to revert it :woman_facepalming: